sourdough/sourdough.py

```diff
@@ -333,6 +333,22 @@ def isCheffed():
   logger.critical('Chef client files found')
   return True
 
+def isDisabled():
+  '''
+  Detect if Chef converge disabled on a system.
+
+  rtype: bool
+  '''
+  logger = this.logger
+  logger.info('Checking for disable switch')
+  disableFile = "/etc/sourdough/Disable-Sourdough"
+  logger.debug("  Checking for %s", disableFile)
+  if os.path.isfile(disableFile):
+      logger.debug("  %s found", disableFile)
+      logger.critical('Chef converge disabled')
+      return False
```

Comment:
Return `True` if the disable is found and false otherwise. Returning `False` feels confusing.

Reply:
@unixorn changed in 811c1cc please review, I concentrated on syntax aesthetic and completely forgot logic.